website/blog/2024-06-24-release-1.11.md

```diff
@@ -0,0 +1,249 @@
+---
+title: Podman Desktop 1.11 Release
+description: Podman Desktop 1.11 has been released!
+slug: podman-desktop-release-1.11
+authors: cdrage
+tags: [podman-desktop, release, podman]
+hide_table_of_contents: false
+image: /img/blog/podman-desktop-release-1.11/light-mode.png
+---
+
+import ReactPlayer from 'react-player'
+
+Podman Desktop 1.11 Release! 🎉
+
+![Podman-desktop-1-11-hero](img/podman-desktop-release-1.11/banner.png)
+
+This release introduces:
+
+- **Light mode!**: Our most-requested feature is here! Try out our new experimental light mode in the settings.
+- **macOS Rosetta support for Apple Silicon**: Build AMD64 binaries at near-native speeds as ARM64 builds.
+- **Kubernetes improvements**: Check out our new Node and Volume pages for Kubernetes.
+- **Improved UI**: In addition to our light mode, you'll notice updates to our container listing page.
+- **Increased manifest support**: You'll now see images grouped together if built with a manifest.
+
+Podman Desktop 1.11 is now available. [Click here to download it](/downloads)!
+
+<!--truncate-->
+
+---
+
+## Release Details
+
+### Light Mode 💡
+
+We're excited to announce the arrival of our most-requested feature.. light mode! You can now enjoy a brighter interface by toggling the light mode option in the settings section. Please note, this feature is currently experimental as we continue to optimize the rest of Podman Desktop for this new mode.
```

Comment:
Should we put 'Experimental' in the title, say 'almost here', or 'imminent arrival'? It feels a little like we're announcing it, then later mentioning that it's not quite done/fully here.

'..' looks like a typo, I'd go the full '...' or use '-' or ':' instead.

Settings should be capitalized, we often put the fa gear in too. Could also say the name of the setting or include a screenshot.

Maybe expand a little bit and mention this will help people with astigmatism or say thank you for your patience?